University-advisor-web/Models/UniversityModel.cs

```diff
@@ -18,11 +18,43 @@ public class UniversityModel
         public string Cost { get; set; }
         public string Image { get; set; }
 
+        public int RankCountry { get; set; }
+        public int RankWorld { get; set; }
+        public string ControlType { get; set; }
+        public string EntityType { get; set; }
+        public string AcademicCalendar { get; set; }
+        public string CampusSetting { get; set; }
+        public string ReligiousAffiliation { get; set; }
+        public string Library { get; set; }
+        public string Housing { get; set; }
+        public string SportFacilities { get; set; }
+        public string FinancialAids { get; set; }
+        public string StudyAbroad { get; set; }
+        public string DistanceLearning { get; set; }
+        public string FacebookLink { get; set; }
+        public string TwitterLink { get; set; }
+        public string LinkedinLink { get; set; }
+        public string YoutubeLink { get; set; }
+        public string InstagramLink { get; set; }
+        public string WikipediaLink { get; set; }
+        public string Acronym { get; set; }
+        public string Founded { get; set; }
+        public string MottoNative { get; set; }
+        public string MottoEnglish { get; set; }
+        public string TelephoneNum { get; set; }
+        public string FaxNum { get; set; }
+        public string LocalUndergraduateCost { get; set; }
+        public string InternationalUndergraduateCost { get; set; }
+        public string LocalPostgraduateCost { get; set; }
+        public string InternationalPostgraduateCost { get; set; }
+
+
         public UniversityModel(){}
```

Comment:
Even if constructor is empty make it same style as others.
`public UniversityModel()
{

}`

Reply:
Will fix that